### DISCUSSION TITLE
Omega Template Documentation no longer available

### DISCUSSION THREAD

**Discussion by ismart-dev** [primefaces/community#1515] [PrimeFaces Templates] [answered]
I recently started a POC to integrate the latest version of the Primefaces Omega template into my app. I followed the guidelines on the following web page:

https://primefaces.org/omega/documentation.xhtml

However, it now seems this web page is no longer accessible and results in a 404 error. 

In addition, looking at the available templates for Primefaces here:

https://www.primefaces.org/store/templates.xhtml

Omega no longer appears in the list (it did a couple of weeks ago).

So questions:

(1) Is Omega an ongoing supported template or has it now been deprecated/removed? It would be slightly disappointing if it's been removed as it let me buy it just a couple of weeks ago.
(2) Can the documentation be reinstated for those of us who may still wish to proceed with Omega?

Thanks in advance for any help/pointers.

**Comment by melloware** [comment]
Let me ask PrimeTek it seems a few of the the templates have been removed.

**Reply by rider87** [reply]
Did you got a reply? So many complaints and no reaction from PrimeTek? Why?

**Comment by kwee27** [comment]
I'm also looking for the rain template ... seems to have also disappeared.

**Reply by melloware** [reply]
It looks like all the templates have disappeared except for a handful. These are the only templates still alive with Showcases:

![image](https://github.com/primefaces/community/assets/4399574/0c898d53-6015-41da-9b22-019c021e4a9f)

Hopefully PrimeTek will chime in with what is going on.

**Comment by persdv** [comment]
I'm another license-owner of Omega template. 
Does this mean that omega-ng template has been discontinued? 
Some information about it would be welcomed...

**Comment by rider87** [comment]
Guys, what´s the status about those missing / removed templates ?????????????
I see so many complaints here and no reaction from PrimeTek! No answer in the forum, no answer in discord.
I´m working with Primefaces more than 10 years.

What´s wrong?

**Comment by melloware** [comment, marked as answer]
Please see: https://github.com/orgs/primefaces/discussions/1806

**Comment by tandraschko** [comment]
we decided to only maintain following templates, which were released yesterday:

          - "apollo"
          - "atlantis"
          - "avalon"
          - "diamond"
          - "freya"
          - "poseidon"
          - "ultima"
          - "verona"
